packages/trader/src/AppV2/Components/RiskManagementItem/risk-management-item.tsx

```diff
@@ -122,7 +122,11 @@ const RiskManagementItem = observer(
                         inputSize='md'
                         disabled={isSheetOpen}
                         textAlignment='center'
-                        value={`${finalValue.toFixed(2)} ${currency}`}
+                        value={`${
+                            type == 'stop_loss'
+                                ? `-${finalValue.toFixed(2)} ${currency}`
```

Comment:
I think we might have missed out these parts before but if the currency is let's say BTC, it shouldn't switch to fixed number of decimals of 2. The number should be based on the currency.

Reply:
please copy the formatMoney from shared to AppV2 utils folder

Reply:
I am using the formatMoney function from shared package now, thanks